### DISCUSSION TITLE
External Images

### DISCUSSION THREAD

**Discussion by ChrisRisner** [nunocoracao/blowfish#780] [General]
I'm using BlowFish for a site with a GitHub repo.  I'm placing on having quite a few images and I'm looking at leveraging an external source for images (blob storage on azure probably) so my repo doesn't become bloated.  Doing so seems to negate the benefits of Hugo and BlowFish's image processing.  Are there any best practices on how to do this?  I've noticed that if I try to use a few of the features (ie carousel) with external images, they load pretty slowly because they aren't being optimized.

**Comment by nunocoracao** [comment]
Hello @ChrisRisner sounds like a great addition to Blowfish. Feel free to open a new feature request so that I can take it on when I have time. Multiple users have been asking for this in this repo (also directly in Hugo I think) - AFAIK no obvious solution.
